src/CoreWCF.Primitives/src/CoreWCF/Security/MessageSecurityOverHttp.cs

```diff
@@ -37,6 +39,15 @@ public MessageCredentialType ClientCredentialType
                 {
                     throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentOutOfRangeException("value"));
                 }
+                if(value is MessageCredentialType.Windows)
+                {
+                    //TODO Remove this after .net 5+
+                   var frameworkDescription = RuntimeInformation.FrameworkDescription;
+                    if (frameworkDescription.IndexOf(NetFrameworkFrameworkName, StringComparison.Ordinal) >= 0)
```

Comment:
Nit: Formatting. Missing blank line after closing braces and indentation is wrong.

Reply:
done